src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java

```diff
@@ -209,9 +215,8 @@ public void initialize(BatchRuntimeContext context) throws Exception {
     super.initialize(context);
     FailureCollector collector = context.getFailureCollector();
     config.validate(collector);
-    Schema schema = config.getSchema(collector);
```

Comment:
what is the current behavior if user only selects a,b,c columns to be written from record of a,b,c,d? would this code change modify current behavior?

Reply:
Good catch, fixed